starter-code/public/scripts/controllers/articleController.js

```diff
@@ -5,7 +5,7 @@ var app = app || {};
   const articleController = {};
 
   // COMMENT: What is this function doing? Where is it called? Does it call any other functions, and if so, in what file(s) do those function(s) live?
-  // (put your response in a comment here)
+  // this gets called in routes and runs articleView, passing ctx of articles.
   articleController.index = (ctx) => app.articleView.index(ctx.articles);
```

Comment:
> passing ctx of articles

I don't quite get what you mean by this. What is `ctx` and why does it have `articles`?